lib/terraform/runner.rb

```diff
@@ -180,6 +188,26 @@ def encoded_zip_from_directory(template_path)
         end
       end
 
+      # Parse Variables in Terraform Template
+      def template_variables(
+        template_path
+      )
```

Comment:
This isn't a very long argument list :) can we one-line this?
```suggestion
      def template_variables(template_path)
```